src/styles/globals.css

```
@@ -0,0 +1,651 @@
+  --h-primary: 180deg; --s-primary: 38%; --l-primary: 29%;
+  --h-secondary: 252deg; --s-secondary: 40%; --l-secondary: 65%;
+  --h-tertiary: 45deg; --s-tertiary: 70%; --l-tertiary: 65%;
+  --h-bg: 43deg; --s-bg: 20%; --l-bg: 95%;
+  --h-heading: 240deg; --s-heading: 8%; --l-heading: 15%;
+  --h-text: 240deg; --s-text: 3%; --l-text: 23%;
+  --h-subtitle: 240deg; --s-subtitle: 3%; --l-subtitle: 43%;
+  --h-outline: 180deg; --s-outline: 38%; --l-outline: 29%;
+  --h-info: 180deg; --s-info: 30%; --l-info: 95%;
+  --h-warning: 45deg; --s-warning: 91%; --l-warning: 49%;
+  --color-outline: hsla(180deg 38% 29% / 0.7);
+  --color-info: var(--color-bg-active);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "--color-info" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)